handler/api/src/main/java/io/knotx/fragments/handler/api/ActionConfig.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (C) 2019 Knot.x Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.knotx.fragments.handler.api;
+
+import io.knotx.fragments.handler.api.actionlog.ActionLogMode;
+import io.vertx.core.json.JsonObject;
+
+public class ActionConfig {
```

Comment:
I like the idea that we have some ActionConfig!

Actions right now are both:
- simple actions that operate on Fragments (`F -> (F',T)`)
- behaviours which point to simple actions (the `doAction` parameter)

The `doAction` parameter is the separate option in our action configuration:
```
behaviour-action {
  factory = "behaviour-factory-name"
  config {
    ...
  }
  doAction = product
}
```
Now we are introducing the `actionLog` parameter in the same way.

However, in the code we specify `ActionConfig` that stores both the Action configuration and the tracing mode.

Maybe, we can move the `doAction` parameter to `ActionConfig` and then update our configurations to:
```
behaviour-action {
  factory = "behaviour-factory-name"
  config {
    doAction = product
    tracing {
      mode = INFO
    }
    options {
      ...
    }
  }
}
```
With this approach the `config` entry is clear - ActionConfig. Adding more ActionConfig parameters in the future will not break the Action Factory interface.

Reply:
we decided offline that no changes in configuration